src/pages/stories/[...slug].js

```diff
@@ -0,0 +1,112 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import ExpandedStoy from "@/pesayetu/components/ExpandedStoy";
+import Hero from "@/pesayetu/components/OtherHero";
+import Page from "@/pesayetu/components/Page";
+import formatBlocksForSections from "@/pesayetu/functions/formatBlocksForSections";
+import getPostTypeStaticPaths from "@/pesayetu/functions/postTypes/getPostTypeStaticPaths";
+import getPostTypeStaticProps from "@/pesayetu/functions/postTypes/getPostTypeStaticProps";
+
+// Define route post type.
+const postType = "post";
+const dateOptions = {
+  year: "numeric",
+  month: "long",
+};
+
+export default function Index({ archive, post, blocks, ...props }) {
+  const { author, categories, featuredImage, date } = post;
+  let authorName = `${author?.node?.firstName ?? ""} ${
+    author?.node?.lastName ?? ""
+  }`;
+  if (authorName?.length < 2) {
+    authorName = author?.node?.nickname ?? author?.node?.slug;
+  }
+  const image = featuredImage?.node?.sourceUrl;
+
+  const category = categories?.edges[0]?.node?.name;
+
+  return (
+    <Page {...props}>
+      {archive ? (
+        <Hero {...blocks?.otherHero} />
+      ) : (
+        <ExpandedStoy
```

Comment:
Wouldn't _StoryPage_ be a better name than _ExpandedStoy_ (which should be _ExpandedSto**r**y_ I assume)